masonry_core/src/passes/event.rs

```diff
@@ -21,7 +21,9 @@ fn get_pointer_target(
 ) -> Option<WidgetId> {
     // See the [pointer capture documentation](../doc/06_masonry_concepts.md#pointer-capture).
     if let Some(capture_target) = root.global_state.pointer_capture_target {
-        return Some(capture_target);
+        if root.is_still_interactive(capture_target) {
```

Comment:
I'd expect this to be a "does exist" check, rather than an "is interactive" check, because of:
https://github.com/linebender/xilem/blob/e1603483aa7117e43cd7b0038450a86f5e5ac209/masonry_core/src/passes/update.rs#L739-L745

Either that, or I'd expect the linked code to also change in this PR.

I think making this a "does exist" check should be sufficient, because the rewrite passes all run between events.

Reply:
I see your point. I'm a bit worried that removing that check means we might still send events to non-interactive widgets for a frame before the event pass runs, but I don't see an obvious way to deal with it, and it should be rare in practice.

Reply:
> I see your point. I'm a bit worried that removing that check means we might still send events to non-interactive widgets for a frame before the event pass runs, but I don't see an obvious way to deal with it, and it should be rare in practice.

I don't think it's for a frame; that event pass happens "immediately", so it's only until the `run_update_pointer_pass` runs, which should happen after every "real" event anyway.